### PR TITLE
Update OpenVINO version

### PR DESCRIPTION
**Pull request by KodiaqQ**
### Changes

- Updated OpenVINO version to 23.1

### CODE REVIEW

**Review by alexsu52 — CHANGES_REQUESTED** [review]
Please update openvino and openvino-dev version in all NNCF.

**Comment by alexsu52** [comment]
run: test_examples/92/

File: examples/post_training_quantization/tensorflow/mobilenet_v2/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 tensorflow~=2.12.0
 tensorflow-datasets
 tqdm
-openvino-dev==2023.0.1
+openvino-dev==2023.1
```
Comment (alexsu52) [inline examples/post_training_quantization/tensorflow/mobilenet_v2/requirements.txt:4]:
```suggestion
openvino-dev==2023.0.2
```
take into account test_examples/92/ results

**Review by alexsu52 — CHANGES_REQUESTED** [review]

File: examples/post_training_quantization/tensorflow/mobilenet_v2/requirements.txt (hunk 1 above)
Comment (KodiaqQ) [reply examples/post_training_quantization/tensorflow/mobilenet_v2/requirements.txt]: Build - 93

**Review by alexsu52 — APPROVED** [review]

**Pull request merged**